WakaTime/WakaTimeConfigFile.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Text;
+
+namespace WakaTime
+{
+    class WakaTimeConfigFile
+    {
+        internal string ApiKey { get; set; }
+        internal string Proxy { get; set; }
+        internal bool Debug { get; set; }
+
+        private readonly string _configFilepath;
+
+        internal WakaTimeConfigFile()
+        {
+            _configFilepath = GetConfigFilePath();
+            Read();
+        }
+
+        internal void Read()
+        {
+            var ret = new StringBuilder(255);
```

Comment:
Some config values could be longer than 255 characters, let's use 2083 (max url length).